.github/workflows/build.yml

```diff
@@ -118,6 +120,9 @@ jobs:
       with:
         ruby-version: 3.2
 
+    - run: chmod 755 $HOME # https://github.com/actions/virtual-environments/issues/267
```

Comment:
Do we need this? The Rubies built by setup-ruby remove this warnings so it shouldn't happen.

Reply:
It was already there, I was just moving it around. 755 _does_ seem like the wrong permission for $HOME...

I'll try removing it in a separate PR perhaps.

Reply:
Ah right, it was not clear in the diff. I think it can be removed, I'll try it.